client/src/Components/Card/index.js

```diff
@@ -80,7 +85,27 @@ export default function CardComponent({
         </CardContent>
       </CardActionArea>
       <CardActions className={classes.cardAction}>
-        <Button className={classes.border}>more details</Button>
+        <Link
+          to={{
+            pathname: `/houses/${id}`,
```

Comment:
You could use your path constants to create this pathname to make sure they stay in sync